home.nix

```diff
@@ -0,0 +1 @@
+path: ~/. + path
```

Comment:
Why is this in a separate file?

Reply:
Oh wow, I didn't realize that the check is more or less at the syntax level. That's strict.